src/components/Header.js

```diff
@@ -0,0 +1,8 @@
+export default function Header() {
+  return (
+    <header className="header-wrapper">
```

Comment:
I like that you wrapped the entire header in a `header-wrapper` class so you can control margins and position in case future needs require